src/tools/roman-numeral-converter/roman-numeral-converter.vue

```diff
@@ -2,21 +2,27 @@
   <div>
     <n-card title="Arabic to roman">
       <n-space align="center" justify="space-between">
-        <n-input-number v-model:value="inputNumeral" :min="1" style="width: 200px" :show-button="false" />
+        <n-form-item v-bind="validationNumeral">
+          <n-input-number v-model:value="inputNumeral" :min="1" style="width: 200px" :show-button="false" />
+        </n-form-item>
         <div class="result">
-          {{ outputRoman }}
+          <span v-html="outputRoman"></span>
```

Comment:
I'm not a fan of v-html, it can lead to a huge security flaw, and is often synonyme of bad design